lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
> I can't see an added benefit in making the method explicit in the spec. 

I do think that there is value in explicitly documenting potentially state changing backend requests. Most end points are GET only, so in most cases we don't need to mention the method & can rely on the `get` default.